src/main/java/uk/gov/companieshouse/mapper/email/EmailMapper.java

```diff
@@ -7,11 +7,11 @@
 import uk.gov.companieshouse.util.DateTimeGenerator;
```

Comment:
You can remove this, and reuse your static import below

Reply:
removed